megamek/src/megamek/client/ui/swing/lobby/MekForceTreeRenderer.java

```diff
@@ -134,7 +134,7 @@ private void setIcon(Image image, int height) {
             int width = height * image.getWidth(null) / image.getHeight(null);
             setIcon(new ImageIcon(ImageUtil.getScaledImage(image, width, height)));
         } else {
-            logger.error("Trying to resize a unit icon of height or width 0!");
+            logger.debug("Trying to resize a unit icon of height or width 0!");

```

Comment:
an error should be something that we need to fix, this apparently is here just to tell "hey, the image is not imaging"